snooty/parser.py

```diff
@@ -940,6 +943,18 @@ def validate_and_add_asset(
                 CannotOpenFile(Path(image_argument), err.strerror, line)
             )
 
+    def validate_relative_url(self, url_argument: str, line: int) -> None:
```

Comment:
Just chewing on this -- we have a *horribly* named `util.add_doc_target_ext()` function that might do what's needed here? It's invoked by `Project.validate_doc_role()`, which is pretty similar to this use case I think.

cc @rayangler

Reply:
That's a good point. I forgot that function existed 😅 Yeah, they do similar things so leveraging that might be better in this case